WcaOnRails/spec/jobs/sync_mailing_lists_job_spec.rb

```diff
@@ -71,6 +71,8 @@
     wst_admin_member = FactoryBot.create :user, :wst_admin_member, team_leader: false
     wac_member = FactoryBot.create :user, :wac_member, team_leader: false
     wac_leader = FactoryBot.create :user, :wac_member, team_leader: true
+    wsot_member = FactoryBot.create :user, :wst_member, team_leader: false
+    wsot_leader = FactoryBot.create :user, :wst_member, team_leader: true
     expect(GsuiteMailingLists).to receive(:sync_group).with(
       "leaders@worldcubeassociation.org",
       a_collection_containing_exactly(wrt_leader.email, wdc_leader.email, wfc_leader.email),
```

Comment:
Since you're creating a `wsot_leader` above, you have to consider he will appear in the `leaders@` mailing list. This test has to be adjusted accordingly.